samples/samples-python/requirements.txt

```diff
@@ -1,3 +1,3 @@
 # Do not include azure-functions-worker as it may conflict with the Azure Functions platform
 
-azure-functions==1.11.3b1
\ No newline at end of file
+azure-functions
```

Comment:
There's a couple other places we reference this - should remove those too right? And then do we have this in the Learn docs anywhere?

Reply:
Sounds good, thanks Charles!